meterfs/common.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs tests common part
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ *
+ * %LICENSE%
+ */
+
+#include "common.h"
+
+static struct {
+	char buff[40];
+}common;
+
+void common_fileInfoFill(file_info_t *info, size_t sectors, size_t filesz, size_t recordsz, size_t recordcnt)
```

Comment:
I don't see real benefit from this function. In most cases you could assign variable while declaration or use compound literal if needed.

Reply:
Done.